requirements/base.txt

```diff
@@ -6,7 +6,7 @@ django-mptt==0.14.0 ; python_version >= "3.10" and python_version < "4.0"
 django-widget-tweaks==1.4.12 ; python_version >= "3.10" and python_version < "4.0"
 django==4.1.2 ; python_version >= "3.10" and python_version < "4.0"
 factory-boy==3.2.1 ; python_version >= "3.10" and python_version < "4.0"
-faker==15.1.0 ; python_version >= "3.10" and python_version < "4.0"
+faker==15.1.1 ; python_version >= "3.10" and python_version < "4.0"
```

Comment:
post it : deplacer `factory-boy` et `faker` dans les lib de dev dans une prochaine PR